docker-compose.yml

```diff
@@ -207,7 +205,6 @@ services:
     volumes:
       - ./app:/usr/src/code/app
```

Comment:
We need to add the `_APP_MAINTENANCE_RETENTION_CACHE` variable to all the containers that need it. 

It includes the `appwrite` and `appwrite-maintenance` containers